src/test_l1_dsHost.c

```diff
@@ -258,9 +260,11 @@ void test_l1_dsHost_negative_dsHostTerm(void) {
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
 	
     // Step 01: dsHostTerm() Attempt to terminate dsHost without initialization
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     dsError_t result = dsHostTerm();
     UT_ASSERT_EQUAL(result, dsERR_NOT_INITIALIZED);
```

Comment:
Comment only the error code, not the API. Also, add dsERR_NONE, which is the expected error code, in the else case.

Reply:
Done